app/Http/Controllers/UsersController.php

```diff
@@ -448,13 +448,34 @@ public function scores($_userId, $type)
      *
      * See [Get User](#get-user).
      *
+     * Additionally, `statistics_fruits`, `statistics_mania`, `statistics_osu`, and `statistics_taiko` are also included.
```

Comment:
I think it's included in the form of `statistics_rulesets`? So something like:
```suggestion
     * Additionally, `statistics_rulesets` is included, containing statistics for all rulesets.
```